fs2-aws-dynamodb/src/main/scala-3.0-/fs2/aws/dynamodb/parsers/package.scala

```diff
@@ -39,10 +40,12 @@ package object parsers {
 
   def parseDynamoRecord[F[_]: Sync, T](
       image: util.Map[String, AttributeValue]
-  )(implicit df: DynamoFormat[T]): F[T] =
+  )(implicit df: DynamoFormat[T]): F[T] = {
+    val v2Image = toAttributeValueMapV2(image)
```

Comment:
arguably, we should just drop dynamodb streaming support if AWS doesn't care about it in SDK v2